apps/src/maze/maze.js

```diff
@@ -181,17 +181,11 @@ module.exports = class Maze {
       });
     };
 
-    // Note: Setting alwaysHideRunButton in our config is necessary because
-    // StudioApp.js will manipulate the run/reset buttons, displaying the run
-    // button when "Reset" is clicked. We do not want this behavior on stepOnly levels.
-    const alwaysHideRunButton = !!(
-      this.controller.level.stepOnly && !this.controller.level.edit_blocks
-    );
-    config.alwaysHideRunButton = alwaysHideRunButton;
```

Comment:
removing this config option that i introduced in the original PR as it's no longer necessary because we honor `hideRunButton`, which is set in redux below